.travis.yml

```diff
@@ -33,16 +33,19 @@ matrix:
     include:
         - os: linux
           compiler: clang-3.6
-          env: CONFIG_OPTS="-fsanitize=address no-shared"
+          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 -DPEDANTIC"
```

Comment:
No, because I can't get shared asan builds to work with clang (https://github.com/google/sanitizers/wiki/AddressSanitizerAsDso) - and in fact am giving up on shared asan + gcc for the moment as well, because of other problems with our wrapper scripts.

So this PR now just runs shared ubsan builds. (Commit description updated to reflect reality.)